resizeobserver/demo.js

```diff
@@ -0,0 +1,13 @@
+/* global ResizeObserver */
+
+const ro = new ResizeObserver(entries => {
```

Comment:
I switched to changing the background image. WDYT @atotic? This sample is more about showing usage and I wanted to keep it small. I’ll rework the sample for the `/updates` article.


<img width="527" alt="screenshot 2016-10-08 09 43 22" src="https://cloud.githubusercontent.com/assets/234957/19211733/b10dc2f0-8d3b-11e6-9633-d313e5aa1b15.png">